### DISCUSSION TITLE
Transaction support

### DISCUSSION THREAD

**Discussion by hammeronthenet** [alexandrainst/node-red-contrib-postgresql#57] [Q&A]
Hi, 
not properly an issue but i don't understand if it's possible in some way to use transaction with automatic rollback if an error occoured.

If yes, how/where to find some example/documentation about that?

Thanks in advance.

**Comment by Alkarex** [comment]
Hello,
You can write any SQL statement you want. Have you tried something like the following?

```sql
BEGIN;
INSERT INTO products (product_no, name, price) VALUES (1, 'Cheese', 9.99);
COMMIT;
```

See https://www.postgresql.org/docs/current/tutorial-transactions.html
